src/main/python/main/ayab/engine/options.py

```diff
@@ -122,14 +123,15 @@ def update_needles(self) -> None:
         stop_needle = NeedleColor.read_stop_needle(self.ui, self.machine)
         self.emit_needles_updater(start_needle, stop_needle)
 
-    def __reverse_image(self) -> None:
-        if self.ui.auto_mirror_checkbox.isChecked():
+    def __auto_mirror_changed(self) -> None:
+        reversed = self.ui.auto_mirror_checkbox.isChecked()
```

Comment:
Would you mind to use another name for this variable, as the current name is already taken by a Python builtin method? (see SonarQube analysis)

Reply:
Ooh, good catch. I'll remember to check the sonar analysis in the future.

I've gone ahead and renamed all occurrences of `reversed` to `image_reversed`.